components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -390,12 +390,20 @@ impl Interpolate for BackgroundSize {
 impl Interpolate for RGBA {
     #[inline]
     fn interpolate(&self, other: &RGBA, progress: f64) -> Result<Self, ()> {
```

Comment:
Can you add a link to https://drafts.csswg.org/css-transitions/#animtype-color on top of the function signature?